.github/workflows/pr.yml

```diff
@@ -3,7 +3,7 @@ name: Pull Request
 on:
   pull_request:
     types: [opened, synchronize]
-    branches:
+    branches: # Target Branches
       - main
       - release/*
       - release-next
```

Comment:
I kinda just made up that branch name one day... guess it's been codified now 😁

Reply:
:trogdor: